### ISSUE TITLE
Resource Type Filter - 2.4.7 Focus Visible - Arrow keys move the user into the Reousrce Type List, but no focus indicator is shown

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9985]
As a user reliant on the keyboard for navigating through a site, focus indicators should accurately reflect the user's location (i.e. which component they are focusing on) within the page.

**Linked Work Item:**
#9493 

**Expected Result:**
User should be able to use either tab or arrow keys, or both, to move into a list, and the focus indicator should highlight which list item the user is focusing on.

**Actual Result:**
Moving the arrow keys into the Resource Type list causes the screen reader to read out each list item, but the focus indicator remains on the Resource Type button even when scrolling through the list. The focus indicator only flashes up briefly when the user selects a list item, not knowing which item they have selected. This error is not present when the user uses the tab key to move through the list.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/8903e207-4ff6-43b1-bc58-1dd998c74f49)

**Comment by apeters** [comment]
@imogenroberts this seems to be fixed in the latest.  Can you retest?

**Comment by imogenroberts** [comment]
RETEST: Focus indicator now appears on the Resource Type Filter in the same way as all other components on the page.

**Evidence:**
![image](https://github.com/archesproject/arches/assets/77161085/bf5c8d2e-79b1-4a9f-8b7a-ad3d180a667e)
